docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc

```diff
@@ -0,0 +1,28 @@
+[[dcb-requisition]]
+= Create Requisition for Device Configuration Backup
+
+After you have xref:operation:device-config-backup/ssh.adoc#backup-script[created a backup script for each device type] and xref:operation:device-config-backup/configuration.adoc#poller-config[configured the TFTP port], you can create a requisition for each device whose configuration you want to back up.
+
+To activate the device configuration backup, you must add metadata that defines the device configuration backup script you created for the node's device type, and the device's username and password.
+
+. Log in to OpenNMS and click the gears icon.
+. Choose *Provisioning>Manage Provisioning Requisitions*.
+. Click *Add Requisition* and enter a name for the requisition.
+. Click on the edit icon and click *Add Node*.
+. Set up your requisition as described in xref:operation:provisioning/getting-started.adoc#requisition-create[Create a requisition].
+. Click *Add Meta-Data*, and define the following key-value pairs:
+.. dcbusername: username for the device
+.. dcbpassword: password for the device
+.. script-file: name of the device configuration script for this type of device
+. Click *Save* to save the metadata and then *Save* to save the requisition.
+
+[[dcb-backup]]
+== Run a device configuration backup
+Since no device configuration back ups will appear in the UI until one runs, we recommend that you run a requisition immediately.
```

Comment:
```suggestion
Since no device configuration backups will appear in the UI until at least one has been collected, we recommend that you synchronize the requisition immediately.
```